modules/network/nsg-bastion.tf

```diff
@@ -40,7 +42,52 @@ locals {
         protocol = local.tcp_protocol, port = local.apiserver_port, destination = local.control_plane_nsg_id, destination_type = local.rule_type_nsg,
       },
     } : {},
-  ) : {}
+  )
+
+  bastion_stateless_rules = merge(
```

Comment:
Bastion and operator are used for admin/troubleshooting purposes. I don't see the need to make their rules stateless.